test/extensions/tracers/zipkin/span_buffer_test.cc

```diff
@@ -211,21 +223,23 @@ TEST(ZipkinSpanBufferTest, ConstructBuffer) {
 TEST(ZipkinSpanBufferTest, SerializeSpan) {
```

Comment:
Is there a way to create a full body of expectations of the serialized span per zipkin spec so we don't encounter missing fields like this PR in the future?

Reply:
Will do. I'll take a look at how to make this test simpler.